documentation/disaster-recovery.md

```diff
@@ -0,0 +1,59 @@
+# Disaster recovery
+
+## Data loss
+
+In the case of a data loss, we need to recover the data as soon as possible in order to resume normal service.
+The application's database is a postgres instance, which resides on PaaS. This provides a point-in-time backup with
+the resolution of 1 second, available between 5min and 7days ago - this is not applicable to postgres `tiny` plans. We can use
+terraform to create a new database using a point-in-time backup of the affacted database instance.
+
+### Make note of database failure time
+
+Make note of the time the database failure occurred, and then use this to calculate when the integrity of the data in the database was still viable. For instance,
+if data loss or corruption happened at 1200hrs, workout what snapshot time is best for the product. This would determine the value of `SNAPSHOT_TIME` env.
```

Comment:
```suggestion
if data loss or corruption happened at 1200hrs, use this to work out what snapshot time is best for the product (consult with the PM if you are unsure what would be best from the product perspective). This would determine the value of `SNAPSHOT_TIME` env.
```